industry_benchmarks/input_structures/prepared_structures/water_set/urokinase/PREPARATION_DEATILS.md

```diff
@@ -0,0 +1,12 @@
+
+protein:
+ - chainA: cap N- and C-termini
```

Comment:
If the capping was done with maestro, could you add the version number here?

Reply:
done